### DISCUSSION TITLE
title resize on mobile phone

### DISCUSSION THREAD

**Discussion by davyvandelaar** [Lednerb/bilberry-hugo-theme#516] [General]
Hi,
The title in the header of my blog-site is being cut-off on a mobile phone when you hold it in normal portrait mode. My blog is called 'CodeCrusaders' and it works fine on a computer screen. On my phone however it will show 'CodeCrusade', where the 'e' is being cut off.

Is there a way to control this behaviour? Preferred would be automatic resizing ofcourse.
I added a screenshot from my phone to illustrate.

![mobile_title](https://user-images.githubusercontent.com/45564157/226377288-5b014c92-7e7d-45da-a778-1071e44362ed.jpg)

**Comment by igor-baiborodine** [comment]
@davyvandelaar, could you please provide me with a link to your website?

**Reply by davyvandelaar** [reply]
ofcourse, I am still developing and experimenting. So, it is mostly still the ExampleSite. Thanks for looking into this.
[https://blogger.technokoning.nl/](https://blogger.technokoning.nl/ )

**Reply by davyvandelaar** [reply]
@igor-baiborodine I have been researching a little bit and what I eventually did was changing the **_responsive.scss** file. And i changed it to ` header .container .titles .title font-size: 2.3rem` . It was 3rem and that caused the word to be larger than the container. Most likely because CodeCrusaders is one word and doesn't cut off. The new setting works for me.

If you feel there is a better way to solve this you can recreate the situation with the google chrome developer tools. Use the responsive settings for samsung s8+ and set the fontsize back to 3rem. Else I am okay with closing this discussion.

Thanks for reaching out
